runtime/lua/vim/_editor.lua

```diff
@@ -26,6 +26,8 @@
 -- These are for loading runtime modules lazily since they aren't available in
 -- the nvim binary as specified in executor.c
 for k, v in pairs({
+  _pos = true,
+  _range = true,
```

Comment:
The underscores aren't necessary, we can just document them as experimental. We aren't going to release Nvim 0.12 until we finalize these.